### PR TITLE
add long descrition

### PR DESCRIPTION
**Pull request by jvanhoefer**
Missing a project description on pypi yet.

### CODE REVIEW

**Review by yannikschaelte — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -9,17 +9,24 @@
...
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as f:
+    long_description = f.read()
+
 setuptools.setup(
     name="yaml2sbml",
     version=version,
     author="Jakob Vanhoefer, Marta R. A. Matos",
     author_email="marta.ra.matos@gmail.com",
     description="A small package to convert ODEs specified in "
                 "a yaml file to SBML/PEtab.",
```
Comment (dilpath) [inline setup.py:27]:
```suggestion
                "a YAML file to SBML/PEtab.",
```

File: setup.py [hunk 2]
```diff
@@ -9,17 +9,24 @@
...
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     url="https://github.com/martamatos/yaml2sbml",
```
Comment (dilpath) [inline setup.py:30]:
```suggestion
    url="https://github.com/yaml2sbml-dev/yaml2sbml",
```

**Review by dilpath — APPROVED** [review]
:+1:

**Comment by jvanhoefer** [comment]
@dilpath Good catches!

**Pull request merged**